JPS_Ontology/ontology/ontoplanningregulation/OntoPlanningRegulation.csv

```diff
@@ -0,0 +1,45 @@
+allowsBuildingMassRatio,Object Property,,,PlanningRegulation,BuildingMassRatio,,A relationship defining the allowable building mass ratio that a given planning regulation has provided.,https://www.theworldavatar.com/kg/ontoplanningregulation/,allows building mass ratio
+allowsGrossPlotRatio,Object Property,,,PlanningRegulation,GrossPlotRatio,,A relationship defining the allowable gross plot ratio that a given planning regulation has provided.,https://www.theworldavatar.com/kg/ontoplanningregulation/,allows gross plot ratio
+allowsSiteCoverage,Object Property,,,PlanningRegulation,SiteCoverage,,A relationship defining the allowable site coverage that a given planning regulation has provided.,https://www.theworldavatar.com/kg/ontoplanningregulation/,allows site coverage
+requiresFloorToFloorHeight,Object Property,,,PlanningRegulation,FloorToFloorHeight,,A relationship defining the required floor to floor height of an urban entity which is enforced by a planning regulation.,https://www.theworldavatar.com/kg/ontoplanningregulation/,requires floor to floor height
+requiresPartyWall,Object Property,,,PlanningRegulation,PartyWall,,A relationship defining the required party wall for an urban entity which is enforced by a planning regulation.,https://www.theworldavatar.com/kg/ontoplanningregulation/,requires party wall
+requiresRoadBuffer,Object Property,,,RoadCategory,RoadBuffer,,A relationship defining the required road buffer for an urban entity which is enforced by a planning regulation.,https://www.theworldavatar.com/kg/ontoplanningregulation/,requires road buffer
+requiresSetback,Object Property,,,PlanningRegulation,Setback,,A relationship defining the required setback of an urban entity which is enforced by a planning regulation.,https://www.theworldavatar.com/kg/ontoplanningregulation/,requires setback
+override,Object Property,,,PlanningRegulation,PlanningRegulation,,This relationship denotes the prioritization of a specific planning regulation over others. It aims to accommodate the diverse hierarchy of regulations observed across different cities.,https://www.theworldavatar.com/kg/ontoplanningregulation/,override
+appliesTo,Object Property,,,PlanningRegulation,https://www.theworldavatar.com/kg/ontozoning/Zone UNION https://www.theworldavatar.com/kg/ontoplot/Plot,,This relationship serves as a link to signify the direct relevance and applicability of the planning regulation to that specific urban entity.,https://www.theworldavatar.com/kg/ontoplanningregulation/,applies to
+appliesTo,Object Property,,,LandUseRegulation,https://www.theworldavatar.com/kg/ontozoning/LandUseType,,This relationship serves as a link to signify the direct relevance and applicability of a land use regulation to that specific land use type.,https://www.theworldavatar.com/kg/ontoplanningregulation/,applies to
```

Comment:
You will need to combine the two definitions of `appliesTo` into a single one. Otherwise the domain will be interpreted as the _intersection_ of the two (which is the empty set).